src/components/FullscreenLoadingIndicator.js

```diff
@@ -25,18 +30,42 @@ const defaultProps = {
  * @param {Object} props
  * @returns {JSX.Element}
  */
-const FullScreenLoadingIndicator = (props) => {
-    if (!props.visible) {
-        return null;
+class FullScreenLoadingIndicator extends React.Component {
+    componentDidMount() {
+        if (!this.props.name || !this.props.timeout) {
+            return;
+        }
+
+        Log.info(`[LoadingIndicator] "${this.props.name}" became visible`);
```

Comment:
**suggestion:** Change these up so that the additional log detail is passed as a parameter instead of a string. And just have `Log.info('[LoadingIndicator] Became visible', {additionalLogDetail})` or whatever we want to name that param. 

**opinion based on log search preferences:** Can be slightly simpler to grep without a * pattern and easier to find logs when the content does not change very often.